rspec-mode.el

```diff
@@ -141,12 +143,22 @@
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-vagrant-cwd "~/"
```

Comment:
Do you think there's a way to parse this value from `Vagrantfile` with any reliability?